app/controllers/katello/api/v2/ostree_branches_controller.rb

```diff
@@ -0,0 +1,16 @@
+module Katello
+  class Api::V2::OstreeBranchesController < Api::V2::ApiController
+    apipie_concern_subst(:a_resource => N_("an ostree branch"), :resource => "ostree_branch")
+    include Katello::Concerns::Api::V2::RepositoryContentController
+
+    private
+
+    def resource_class
+      OstreeBranch
+    end
+
+    def filter_by_content_view_filter(filter)
```

Comment:
No but this method is needed for the index call -> look at https://github.com/Katello/katello/blob/master/app/controllers/katello/concerns/api/v2/repository_content_controller.rb#L87